demo/public/locales/en-US/translation.json

```diff
@@ -0,0 +1,4 @@
+{
+	"Welcome to React": "Vitejte v Reactu",
```

Comment:
This is not english (and I'm not sure if we need `en-US` folder

Reply:
Concerning `en-US` folder - it does not need to be there, but if it is not added, then we have a warning with failed localizations ```something went wrong loading 
Array [ "failed parsing /locales/en-US/translation.json to json" ]```